db/fixtures/ae_datastore/ManageIQ/Infrastructure/VM/Provisioning/Placement.class/__methods__/vmware_best_fit_least_utilized.rb

```diff
@@ -6,7 +6,7 @@ def root_folder(host, data_center)
 end
 
 def set_folder(prov, host, vm)
-  host_dc = host.ems_cluster.v_parent_datacenter
+  host_dc = host.v_owning_datacenter.presence || host.ems_cluster.v_parent_datacenter
```

Comment:
There is a known bug with owning_datacenter that @bdunne is working on.  I think if he fixes that issue, then this fix is not needed (though it's good to have the spec anyway, but maybe @bdunne can take that into his PR).

So, once that bug is fixed, you should be able to call owning_datacenter directly (without the v_ prefix), and you wouldn't need the latter `host.ems_cluster.v_parent_datacenter` if you have the owning_datacenter